contributing.md

```diff
@@ -0,0 +1,11 @@
+# Contributing to the CEPC and the Positive Work Environment Community Group
+
+We welcome contributions from the community as we work to develop a positive and inclusive work environment for W3C. 
+
+If you have feedback or questions relating to the Code of Ethics and Professional Conduct or any of our other projects, please log an issue in this repository. Logging issues allows the chairs to review and raise issues in upcoming meetings. We encourage those who have logged issues to join the meetings, so they can participate in the discussion. You can find meeting information on our [calendar page](https://www.w3.org/groups/cg/pwe/calendar).
+
+We ask that participants do not open Pull Requests with changes to the CEPC or other documents unless asked to do so by the chairs or editors. This is to help us keep discussions in issues or meetings where appropriate, and to help prevent merge conflicts. 
+
+If an issue raises an editorial question or suggestion, the chairs and editors will take care of it in a PR which will then be added to an upcoming meeting agenda for the group to review. 
```

Comment:
I don't think this sentence is needed, since the paragraph before it already says not to create a PR unless asked to do so.